instana/instrumentation/django/middleware.py

```diff
@@ -69,6 +68,11 @@ def process_response(self, request, response):
             return response
 
     def process_exception(self, request, exception):
+        from django.http.response import Http404
```

Comment:
nitpick: can we drop this to the top of the file where all the imports are?

Reply:
The idea is that we import this class in a place where we're sure that django is present. Alternatively I could do this at the beginning of `load_middleware_wrapper()` where we are also importing settings. WDYT?

Reply:
No, then it's okay - LGTM 👍